sdk/keyvault/assets.json

```diff
@@ -2,5 +2,5 @@
   "AssetsRepo": "Azure/azure-sdk-assets",
   "AssetsRepoPrefixPath": "rust",
   "TagPrefix": "rust/keyvault",
-  "Tag": "rust/keyvault_6c9f540438"
+  "Tag": "rust/keyvault_faa717aa41"
```

Comment:
Nit: IMO, instrumenting an existing test might've been better to make sure it didn't alter anything. It shouldn't, but that'd help make sure.

Reply:
I'm a big fan of a test doing one thing and one thing only, rather than a test doing two separate unrelated things. instrumenting an existing test means that it would be doing two different things.